traitsui/testing/tester/query.py

```diff
@@ -39,3 +39,13 @@ class DisplayedText:
     Implementations should return a ``str``.
     """
     pass
+
+
+class IsChecked:
+    """ An object representing an interaction to obtain whether a checkbox
+    is checked or empty.
```

Comment:
Not just checkbox can be have "isChecked" state. Other widgets like radio button can have that state. Perhaps instead of saying "checkbox", this should say "checkable widget (e.g. checkbox)"?

"empty" sounds a bit odd, but that could just be me. "is checked or not" seems good enough?